cmd/windup.go

```diff
@@ -74,6 +105,15 @@ func (r *Windup) options() (options command.Options, err error) {
 	if err != nil {
 		return
 	}
+	if r.Rules != nil {
+		err = r.Rules.AddOptions(&options)
+		if err != nil {
+			return
+		}
+		r.Targets = append(
+			r.Targets,
+			r.Rules.foundTargets...)
```

Comment:
**hint**: This is where the addon injects the targets found in rules files into the list of targets.